enterprise_edition/configuration.mdx

```diff
@@ -47,7 +47,71 @@ export API_KEY_HASH_ROUNDS=600000
 Increasing this value enhances security but may impact performance. The default value provides a good balance for most use cases.
 
 
+## Additional API Endpoints
+
+
+
+### Standard Answers
+
+Endpoint: `POST /chat/test-standard-answer`
```

Comment:
we need to remove the test from here, right?

Reply:
Indeed! Making this change now